[DOCS] 7.14.0 Release Notes (#811)
elastic/security-docs: docs/release-notes.asciidoc

:issue: https://github.com/elastic/kibana/issues/
:pull: https://github.com/elastic/kibana/pull/

[discrete]
[[release-notes-7.14.0]]
== 7.14.0

[discrete]
[[features-7.14.0]]
==== Features
* Host isolation allows analysts to isolate hosts from their networks while investigating a potential attack. Analysts can use this feature to respond to malicious activity by containing infected hosts, curbing potential attacks, and preventing lateral movement to other hosts. This feature is supported on Windows and macOS.
* Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404]) ({pull}95014[#95014])({pull}104984[#104984]).
* Adds threat intelligence to alerts ({pull}101553[#101553])({pull}103383[#103383]).

> * New functionality and usability improvements were added to the Osquery Manager integration:

Reviewer: ```suggestion
* Adds new functionality and usability improvements to the Osquery Manager integration:
```

* Paginates long activity logs ({pull}102261[#102261]).
* Validates path values for trusted apps ({pull}99035[#99035]).
* Allows the wildcard symbol in trusted app paths ({pull}97623[#97623]).

[discrete]
[[bug-fixes-7.14.0]]
==== Bug fixes and enhancements
* Adds a merge strategy key to `kibana.yml` and adds additional security keys to the Docker container that Elastic Security previously overlooked ({pull}103800[#103800]).
* Adds an overflow container to the rule name column in the Exceptions table for exceptions that have been assigned to three or more rules ({pull}103377[#103377]).
* Adds the option to select all rules within the Rules table that match the currently selected filter ({pull}100554[#100554]).
* Adds the Threat Intelligence card to the Overview page ({pull}100423[#100423]).
* Introduces the Swimlane connector for rules and cases ({pull}100086[#100086]).
* Enhances the callout that describes missing privileges and feature access ({pull}98125[#98125]).
* Fixes the rule preview issue that occurred if users created a threshold rule that was configured to group the IP data type ({pull}105126[#105126]).

> * Removes the comma delimiter for the `is one of operator` when defining rule exception conditions ({pull}104960[#104960]).

Reviewer: ```suggestion
* Removes the comma delimiter for the `is one of` operator when defining rule exception conditions ({pull}104960[#104960]).
```

* Resolves bug that left outdated validation messages on the action type selection form ({pull}104868[#104868]).
* Fixes the sort logic that didn't work for certain fields within the Rules table ({pull}103960[#103960]).
* Allows activity log scrolling on small screens ({pull}103852[#103852]).

> * Fixes the bug that cause the checkbox value for *Show only threat indicator alerts* from updating properly within the Alerts table ({pull}103746[#103746]).

Reviewer: ```suggestion
* Fixes the bug that caused the checkbox value for *Show only threat indicator alerts* from updating properly within the Alerts table ({pull}103746[#103746]).
```

* Disables the *Load Elastic prebuilt rules and timeline templates* button when pre-built rules are loading ({pull}103568[#103568]).
* Allows users to view the details of a deleted rule ({pull}103491[#103491]).
* Includes actions and responses for endpoints only ({pull}103159[#103159]).

> * Resolves the issue that cause an error message to display if users created rule exceptions with empty fields ({pull}102583[#102583]).

Reviewer: ```suggestion
* Resolves the issue that caused an error message to display if users created rule exceptions with empty fields ({pull}102583[#102583]).
```

* Removes the search bar on the *Activity log* tab ({pull}102550[#102550]).
* Does not show activity log error popups ({pull}102450[#102450]).
* Shows up to one hour of relative time in the activity log when viewing it from the endpoint details flyout. ({pull}102162[#102162]).
* Updates mappings for detection alerts to ECS v1.10.0 ({pull}101680[#101680]).
* Fixes timestamp bugs within source indexes when the formats are not in ISO 8601 format ({pull}101349[#101349]).
* Exposes the EQL query in Kibana logs for detections ({pull}100565[#100565]).

> * Resolved bugs linked to invalid KQL queries ({pull}99442[#99442]).

Reviewer: ```suggestion
* Resolves bugs linked to invalid KQL queries ({pull}99442[#99442]).
```

:issue: https://github.com/elastic/kibana/issues/
:pull: https://github.com/elastic/kibana/pull/

[discrete]
[[release-notes-7.14.0]]
== 7.14.0

[discrete]
[[features-7.14.0]]
==== Features
* Host isolation allows analysts to isolate hosts from their networks while investigating a potential attack. Analysts can use this feature to respond to malicious activity by containing infected hosts, curbing potential attacks, and preventing lateral movement to other hosts. This feature is supported on Windows and macOS.

> * Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404]) ({pull}95014[#95014])({pull}104984[#104984]).

Reviewer: ```suggestion
* Adds malware protection for Linux endpoints. Users can enable Linux malware protection in their policy to receive detection alerts. ({pull}103404[#103404])({pull}95014[#95014])({pull}104984[#104984]).
```
